### PR TITLE
Feat/vp overlay

### PR DESCRIPTION
**Pull request by OzgeYurtsever**

### CODE REVIEW

**Review by emelalkim — APPROVED** [review]

File: src/components/display/displayView.jsx [hunk 1]
```diff
@@ -164,6 +164,7 @@ const mapStateToProps = (state) => {
     otherSeriesAimsList: state.annotationsListReducer.otherSeriesAimsList,
     templates: state.annotationsListReducer.templates,
     lastLocation: state.annotationsListReducer.lastLocation,
+    seriesData: state.annotationsListReducer.seriesData
```
[the remark this answers is not in the document]
Comment (OzgeYurtsever) [reply src/components/display/displayView.jsx:167]: done

File: src/components/display/displayView.jsx [hunk 2]
```diff
@@ -2550,7 +2551,7 @@ class DisplayView extends Component {
   };
 
   render() {
-    const { series, activePort, updateProgress, updateTreeDataOnSave } =
+    const { series, activePort, updateProgress, updateTreeDataOnSave, seriesData } =
```
[the remark this answers is not in the document]
Comment (OzgeYurtsever) [reply src/components/display/displayView.jsx:2554]: done

File: src/components/display/displayView.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (OzgeYurtsever) [reply src/components/display/displayView.jsx:167]: done

File: src/react-cornerstone-viewport/CornerstoneViewport/CornerstoneViewport.js [hunk 3]
```diff
@@ -326,7 +326,7 @@ class CornerstoneViewport extends Component {
    * @memberof CornerstoneViewport
    */
   getOverlay() {
-    const { viewportOverlayComponent: Component, imageIds } = this.props;
+    const { viewportOverlayComponent: Component, imageIds, seriesDesc } = this.props;
```
[the remark this answers is not in the document]
Comment (OzgeYurtsever) [reply src/react-cornerstone-viewport/CornerstoneViewport/CornerstoneViewport.js:329]: done

File: src/react-cornerstone-viewport/ViewportOverlay/ViewportOverlay.js [hunk 4]
```diff
@@ -55,20 +86,26 @@ class ViewportOverlay extends PureComponent {
...
       cornerstone.metaData.get("imagePlaneModule", imageId) || {};
     const { rows, columns, sliceThickness, sliceLocation } = imagePlaneModule;
     const { seriesNumber } = seriesMetadata;
-    const { seriesDescription } =
-      cornerstone.metaData.get("specialSeriesModule", imageId) || {};
+    // const { seriesDescription } =
+    //   cornerstone.metaData.get("specialSeriesModule", imageId) || {};
+
+    const metadata = cornerstoneWADOImageLoader.wadors.metaDataManager.get(imageId);
+    const seriesDescription = metadata['0008103E'].Value[0];
```
[the remark this answers is not in the document]
Comment (OzgeYurtsever) [reply src/react-cornerstone-viewport/ViewportOverlay/ViewportOverlay.js]: done

**Pull request merged**